config/routes.rb

```diff
@@ -4,7 +4,8 @@
   resources :games do 
   member do 
     put 'join'
-  end 
+  end
+  resources :pieces, :only => [:create, :show, :update] 
```

Comment:
If you remove the `create` method from the controller, you can remove it from here as well